sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/RedundantArgumentCodeFixProvider.cs

```diff
@@ -169,11 +169,11 @@ protected override async Task RegisterCodeFixesAsync(SyntaxNode root, CodeFixCon
         }
 
         private static ArgumentListSyntax AddParamsArguments(SemanticModel semanticModel,
-            ICollection<CSharpMethodParameterLookup.ArgumentParameterMapping> paramsArguments, ArgumentListSyntax argumentList)
+            ICollection<SyntaxNodeSymbolSemanticModelTuple<ArgumentSyntax, IParameterSymbol>> paramsArguments, ArgumentListSyntax argumentList)
         {
             var firstParamsMapping = paramsArguments.First();
-            var firstParamsArgument = firstParamsMapping.Argument;
-            var paramsParameter = firstParamsMapping.Parameter;
+            var firstParamsArgument = firstParamsMapping.SyntaxNode;
+            var paramsParameter = firstParamsMapping.Symbol;

```

Comment:
I know this is just some refactoring on this file, but lines 187-192 don't have test coverage. With a "fix the leak" approach, we should add test coverage on code that we touch (if it's easy to do)

Reply:
I really prefer not to spend too much time on the code fix we provide. The few times I had to update them it ended up taking way much time because of a lot of side effects.